.code-samples.meilisearch.yaml

```diff
@@ -664,3 +665,23 @@ settings_guide_pagination_1: |-
       'maxTotalHits': 50
     }
   })
+async_guide_filter_by_date_1: |-
+  client.get_tasks({'afterEnqueuedAt': '2020-10-11T11:49:53.000Z'})
+async_guide_multiple_filters_1: |-
+  client.get_tasks(
+    {
+        'indexUids': 'movies',
+        'types': ['documentAdditionOrUpdate', 'documentDeletion'],
+        'statuses': ['processing'],
+    }
+  )
+async_guide_filter_by_ids_1: |-
+  client.get_tasks({'uids': ['5', '10', '13']})
+async_guide_filter_by_statuses_1: |-
+  client.get_tasks({'statuses': ['failed', 'canceled']})
+async_guide_filter_by_types_1: |-
+  client.get_tasks({'types': ['dumpCreation', 'indexSwap']})
+async_guide_filter_by_index_uids_1: |-
+  client.get_tasks({'indexUids': 'movies'})
+async_guide_canceled_by: |-
+  client.get_tasks({'canceledBy': [9]})
```

Comment:
```suggestion
async_guide_canceled_by_1: |-
  client.get_tasks({'canceledBy': [9, 15]})
```

Reply:
This change was requested here https://github.com/meilisearch/integration-guides/issues/224#issuecomment-1328645858